### ISSUE TITLE
implement StyleCI

### ISSUE THREAD

**Issue by tejasmanohar** [d11wtq/boris#92]
https://styleci.io/

@d11wtq, what do you think about implementing StyleCI? I would myself, but it only allows you to do this on repos you own, not collaborate to.

**Comment by GrahamCampbell** [comment]
> not collaborate to.

It allows all repos you're "admin" on, rather than ones you physically own.

**Comment by tejasmanohar** [comment]
@GrahamCampbell Ah, alright. Can't have multiple admins on personal repos, I believe.

**Comment by GrahamCampbell** [comment]
> Can't have multiple admins on personal repos, I believe.

Nope, you deffo can. I know repos that have like 12 admins on them.

**Comment by GrahamCampbell** [comment]
GitHub let's admins choose "read", "write", "admin" when you're added as a collaborator.

**Comment by tejasmanohar** [comment]
@GrahamCampbell I can't find that option in any of my repos. Would you post a screenshot / tutorial to demonstrate?

>>>You can grant read/write access to collaborators on a personal repository. Repositories owned by an organization can grant more granular access. For more information, see "What are the different access permissions?"
-- from [Adding collaborators to a personal repository](https://help.github.com/articles/adding-collaborators-to-a-personal-repository/)

**Comment by tejasmanohar** [comment]
anyways, putting us back on topic. @GrahamCampbell feel free to email me if you find out, email is on github profile

>https://styleci.io/

@d11wtq, what do you think about implementing StyleCI? I would myself, but it only allows you to do this on repos you own, not collaborate to.

**Comment by GrahamCampbell** [comment]
To be honest, if boris has multiple maintainers, it would make sense to move it to an organisation. We did that a few weeks ago with class preloader. https://github.com/ClassPreloader/ClassPreloader

**Comment by GrahamCampbell** [comment]
https://github.com/borisphp/boris or https://github.com/borisrepl/boris might work. :)

**Comment by tejasmanohar** [comment]
@GrahamCampbell - Makes sense.

Chaining @d11wtq on this :+1:

**Comment by d11wtq** [comment]
I've converted this repository to an organization and you have admin access now. No objections to StyleCI, nope. I'm easy on coding style, just as long as it's consistent, although I do have a dislike for tabs mostly because they render terribly/unpredictably in things like cat/less/the browser.

**Comment by GrahamCampbell** [comment]
> although I do have a dislike for tabs 

StyleCI only supports fixing to spaces atm anyway. :)

**Comment by GrahamCampbell** [comment]
Why would you want tabs? lol

**Comment by tejasmanohar** [comment]
@GrahamCampbell I mean tabs do have their advantages, like being able to set the appearance custom in your environment, but @d11wtq brings up a good point so we will continue to use spaces.

I'll implement this before EOD tmrw.
